### PR TITLE
Feature/jballiu/other changes

### PR DESCRIPTION
**Pull request by edliraallaraj**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -80,3 +80,7 @@ function isGreat(){
     }
 }
 console.log(isGreat());
+console.log(isGreat());
+console.log(isGreat());
```
Comment (edliraallaraj) [inline index.js:84]: Please, call the function with a parameter.

Comment (edliraallaraj) [reply index.js:84]: Done.

Comment (Jballiu) [reply index.js:84]: done

Comment (OlsiBrusha) [reply index.js:84]: o

**Pull request merged**